src/modules/accessChecks.ts

```diff
@@ -284,6 +284,16 @@ export const accessChecks = {
                 return allowed();
             }
 
+            const interviewAccessCheck = accessChecks.interview.readOne(session, section.interview);
+
+            if (interviewAccessCheck.allowed) {
+                return allowed({ hideSectionGradesBySectionIds: [section.id] });
+            }
+
+            if (!interviewAccessCheck.allowed) {
```

Comment:
Second if is unnecesary and prevents the rest of the function from running

Reply:
Thanks, I replace this check below